eng/pipelines/runtimelab/install-emscripten.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Perhaps we can use the same powershell sharing strategy as with the the other scripts?

Reply:
right, have deletes the .cmd/.sh and just left the `ps1`